test/github-issues/7068/issue-7068.ts

```diff
@@ -0,0 +1,190 @@
+import "reflect-metadata";
+import {Category} from "./entity/Category";
+import {Connection} from "../../../src/connection/Connection";
+import {closeTestingConnections, createTestingConnections, reloadTestingDatabases} from "../../utils/test-utils";
+
+describe.only("github issues > #7068", () => {
```

Comment:
Probably excessive, to have a copy of all of the old tests.

Just try access the `EntityMetadata` from `connection` by filtering for one with the correct name as expected with your configration i.e. "category_xyz_closure". Then check it's `columns` property to have columns named "ancestor_xyz_" ...

Reply:
IMHO metadata checks doesn't cover real work. One time i saw `SubjectExecutor` ignores metadata naming. Maybe other parts of code do the same. Are you shure  it's good idea to reduce tests?

https://github.com/typeorm/typeorm/blob/master/src/persistence/tree/ClosureSubjectExecutor.ts#L79